math-libs/BLAS/CMakeLists.txt

```diff
@@ -128,9 +128,9 @@ if(THEROCK_ENABLE_SPARSE)
       -DROCM_PATH=
       -DROCM_DIR=
       -DBUILD_WITH_ROCBLAS=ON
-      -DBUILD_CLIENTS_TESTS=ON
-      -DBUILD_CLIENTS_BENCHMARKS=ON
-      -DBUILD_CLIENTS_SAMPLES=OFF
```

Comment:
The other changes here are NFC, but this allows enabling the previously explicit disabled option. Is that intentional?